index.js

```diff
@@ -26,6 +34,8 @@ function mergeObject(target, source, options) {
 	Object.keys(source).forEach(function(key) {
 		if (!options.isMergeableObject(source[key]) || !target[key]) {
 			destination[key] = cloneUnlessOtherwiseSpecified(source[key], options)
+		} else if (options.customMerge) {
```

Comment:
We can drop this whole conditional here - that `getMergeFunction` function works whether a `customMerge` was set or not, so all we should need here is

```js
} else {
	destination[key] = getMergeFunction(key, options)(target[key], source[key], options)
}
```

Reply:
Thanks for pointing this out; I'm not sure why I left this conditional in, as I think you had eluded to this already, and the code for `getMergeFunction` makes it fairly clear that we don't need to worry about whether or not `customMerge` is specified. I believe I've now pushed a commit which fixes this